src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -234,7 +234,7 @@ async function pollForTokenWithProgress<T>(
         {
             title: localize(
                 'AWS.auth.loginWithBrowser.messageDetail',
-                'Login page opened in browser. When prompted, provide this code: {0}',
+                'Login page opened in browser. Confirm the code matches: {0}',
```

Comment:
This is probably a case where the code should be as prominent as possible in the message. "Login page opened in browser" has more prominence than it should.

```suggestion
                'Confirm code "{0}" from the login page opened in your web browser.',
```